src/it/resources/com/google/checkstyle/test/chapter3filestructure/toolongpackagetotestcoveragegooglesjavastylerule/InputPackageStatement.java

```diff
@@ -0,0 +1,4 @@
+package com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule; // ok
```

Comment:
Please rename it to InputPackageStatementLongName.java
And do suppression by `line="1"`

Another file will have suppression for whole file, and you did it originally.

Reply:
done

https://github.com/checkstyle/checkstyle/blob/30e9e18604403cf6ea1fefb264cbb16c8ff9b4a2/config/checkstyle-non-main-files-suppressions.xml#L63-L64

https://github.com/checkstyle/checkstyle/blob/30e9e18604403cf6ea1fefb264cbb16c8ff9b4a2/config/checkstyle-non-main-files-suppressions.xml#L76-L78